modules/core/src/lib/constants.ts

```diff
@@ -101,12 +108,27 @@ export const UNIT = {
 } as const;
 
 export const EVENTS = {
```

Comment:
Can these map declarations be properly typed now with mjolnir 3? Maybe an opportunity to make a typing pass?

Reply:
Not sure what types we need for this particular object. It is only used by the Deck class and doesn't have anything to do with mjolnir.

Reply:
IMHO, we shouldn't have untyped objects in our code base.